website/docs/d/nsxv_application_group.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxv_application_group"
+sidebar_current: "docs-vcd-data-source-nsxv-application-group"
+description: |-
+  Provides a VMware Cloud Director data source for reading NSX-V Distributed Firewall application groups
+---
+
+# vcd\_nsxv\_application\_group
+
+Provides a VMware Cloud Director NSX-V Distributed Firewall data source used to read an existing application group
```

Comment:
Nit, it looks strange without a dot in this place.

Reply:
Thanks for the change, let's also use dot in this place.

```suggestion
Provides a VMware Cloud Director NSX-V Distributed Firewall data source used to read an existing application group.
```

Reply:
fixed